orm/src/main/java/org/hibernate/search/query/hibernate/impl/FullTextQueryImpl.java

```diff
@@ -390,12 +390,12 @@ protected boolean isNativeQuery() {
 	}
 
 	@Override
-	public Query setEntity(int position, Object val) {
+	public org.hibernate.query.Query<R> setEntity(int position, Object val) {
```

Comment:
It would be simpler because `FullTextQuery` is what we want to implement in this file, while `org.hibernate.query.Query` is something we happen to inherit from indirectly. And that has the same name as another class we already reference in the same file.

Anyway, it's no important enough to argue about it... Let's keep it that way.